systemtest/jpsreport_tests/method_A/test4/04_method_A.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+#---------
+# 6 pedestrians created at frame >= 9
+# ids are discontinuous
+# creation times in sources are 1, 5, 10, 15, 25, 30
+#---------
+import os
+from sys import argv, path
+import logging
+utestdir = os.path.abspath(os.path.dirname(os.path.dirname(os.path.dirname(path[0]))))
+path.append(utestdir)
+path.append(os.path.dirname(path[0])) # source helper file
+from utils import SUCCESS, FAILURE
+import numpy as np
+import subprocess
+should_be_7 = np.array([7, 11, 16, 21, 31, 36])
+should_be_9 = should_be_7 + 2
+
+def runtest(trajfile):
+    logging.info("===== Method A - Flow-NT ===============")
+    data_9_filename = os.path.join('./Output',
+                                   'Fundamental_Diagram',
+                                   'FlowVelocity',
+                                   'Flow_NT_%s_id_9.dat'%trajfile
+                                   )
+    
+    if not os.path.exists(data_9_filename):
+        logging.error("jpsreport did not output results correctly.")
+        exit(FAILURE)
+    data_9 = np.loadtxt(data_9_filename)
+    time_change_9 = data_9[np.nonzero(np.diff(data_9[:, 1]) >0)][:, 0]
+    if np.all(np.abs(time_change_9 - should_be_9) < 0.5):
```

Comment:
Why `< 0.5`?

Reply:
because of the uncertainties related to `fps`. This threshold is acceptable.